src/pages/Details/Details.js

```diff
@@ -1,10 +1,162 @@
 import React from 'react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AiFillStar } from 'react-icons/ai';
+import TotalInfo from './components/TotalInfo';
+import Option from './components/Option';
+import './components/TotalInfo';
+import './Detail.scss';
 
 const Details = () => {
+  const navigate = useNavigate();
+
+  const [productDetails, setproductDeatils] = useState([{}]);
```

Comment:
```suggestion
  const [productDetails, setproductDeatils] = useState([]);
```
초기값은 해당 값이 들어오는 가장 초기의 형태로 구현해주세요!